rollingpin/deploy.py

```diff
@@ -53,9 +53,16 @@ def __str__(self):
 
 class Deployer(object):
 
-    def __init__(self, config, event_bus, parallel, sleeptime, timeout):
+    def __init__(self, config, event_bus, parallel,
+                 sleeptime, timeout, dangerously_fast):
         """
-        :param timeout: command execution timeout
+        :param config: (dict)
```

Comment:
if this is gonna use sphinx style param descriptions, let's go all the way. the type can be in the `:param:` field like so:

https://github.com/reddit/baseplate/blob/master/baseplate/core.py#L172-L180